docs/README.md

```diff
@@ -181,6 +181,22 @@ Sub-commands:
 * `-f`, `--force`: Overwrite existing scenario setup file.
 * `-h`, `--help`: Show the help for the `template` command.
 
+##### Piping stdin
+
+Config file and override file can be piped from stdin. If no config is set stdin is assumed to be the config file, if config file is set, stdin is assumed to be the ovverides file.
```

Comment:
Tiny spelling error: ovverrides

Reply:
Also it's called "override file" in the previous sentence

Reply:
should be fixed now